code/processes/rdb.q

```diff
@@ -192,9 +193,13 @@ reload:.rdb.reload
 
 .lg.o[`init;"searching for servers"];
 
-//check if tickerplant is available and if not exit with error 
-.servers.startupdepcycles[.rdb.tickerplanttypes;.rdb.tpconnsleepintv;.rdb.tpcheckcycles]
-.rdb.subscribe[]; 
+// connects and subscribes to tickerplant only if connectonstart is true
+$[.rdb.connectonstart;
+ [.servers.CONNECTIONS,:.rdb.tickerplanttypes;
```

Comment:
Could you add some indentation here to make it more clear that this is two conditions